liferay-lima/liferay-portal
# LPS-202830 Translate the decided version not the last published version

how to test it:



Create a new Web Content Article, with title and text “My original article”. Publish
Edit this article - change title&text to “My edited article”, Save as Draft
From the Web Content Folder, pick “Translate” to go to the Side-by-Side translation
Observe that you see the published text, though an updated draft is already available



- LPS-202830 Create method to provide the version to the command
- LPS-202830 baseline
- LPS-202830 take into account that the version can come on the request
- LPS-202830 Use the new method with the version to get the last version object
